about.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css" integrity="sha512-XJ3ntWHl40opEiE+6dGhfK9NAKOCELrpjiBRQKtu6uJf9Pli8XY+Hikp7rlFzY4ElLSFtzjx9GGgHql7PLSeog==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Document</title>
+  </head>
+<body>
+  <header class="header">
+    <div class="top">
+      <nav class="login-section">
+        <div class="log-content">
+          <i class="fa-brands fa-facebook-f"></i>
+          <i class="fa-brands fa-twitter"></i>
+
+          <select name="language" id="language">
+            <option value="" selected="selected">Language</option>
+            <option class="language-link">English</option>
+            <option class="language-link">Spainish</option>
+            <option class="language-link">French</option>
+            <option class="language-link">Arabic</option>
+          </select>
+          
+          <a href="" class="page-link">My Page</a>
+          <a href="">LogOut</a>
+        </div>        
+      </nav>
+      <nav class="navbar">
+        <div class="hamburger menuaction" id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <div class="logo " id="">
+          <a href="index.html"><img src="./assets/images/navlogo.svg" alt=""></a>
+        </div>
+        <nav class="menu" id="menu">
+          <ul class="mobile-menu">
+            <li class="nav-item  primary-font "><a href="./about.html" class="nav-link primary-font menuaction">About</a></li>
+            <li class="nav-item  primary-font "><a href="#about" class="nav-link menuaction">Program</a></li>
```

Comment:
- [ ] Let's add `Hom` button on About page, so that user can come back on the main page.
![home](https://user-images.githubusercontent.com/65860678/227678282-3f35d735-a5ed-4dd7-9066-d5a1e26f732a.GIF)